commcare_connect/opportunity/tables.py

```diff
@@ -58,15 +59,26 @@ def show_warning(record):
 
 class UserVisitReviewFilter(FilterSet):
     review_status = ChoiceFilter(choices=VisitReviewStatus.choices, empty_label="All Reviews")
+    user = ModelChoiceFilter(queryset=User.objects.none(), empty_label="All Users", to_field_name="username")
+    visit_date = DateRangeFilter()
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.filters["user"].queryset = User.objects.filter(id__in=self.queryset.values_list("user_id", flat=True))
```

Comment:
Don't you need to pass any org/program filtering on this queryset?

Reply:
This is using the query set passed for the filter initially, so the user choices will be limited to the User Visits passed in the queryset.